kubefwd/sudo-kubefwd.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+OSASCRIPT=$(command -v osascript)
+PKEXEC=$(command -v pkexec)
+KDESUDO=$(command -v kdesudo)
+GKSUDO=$(command -v gksudo)
+KUBEFWD=$(command -v kubefwd)
+RUN_KUBEFWD=$(pwd)/run-kubefwd.sh
+KUBECONFIG=${KUBECONFIG:-${HOME}/.kube/config}
+
+set -euo pipefail
+
+if [[ "$KUBEFWD" == "" ]]; then
+    echo "kubefwd not found. Did you forget to install it?"
+    echo "Run: brew install txn2/tap/kubefwd"
```

Comment:
total linux erasure!

Reply:
homebrew is on linux now!
https://docs.brew.sh/Homebrew-on-Linux

TBH i've noticed more linux-based projects preferring it over `apt`

it's both slightly nicer for end-users (doesn't require sudo) and much nicer for maintainers (because a homebrew tap can be a github repo)